7.2/7.2/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace Area
+{
+
+    class Program
+    {
+
+        public bool Condition(double x, double y)
+        {
+            if (y > 0.5 & y < 1.5 & x < 2)
+                return true;
+            else if (x > 2)
+                return true;
+            else
+                return false;
+        }
```

Comment:
Несоответствие заданию: неравенства должны быть нестрогими, иначе граница не войдет в область (такая граница на чертеже обозначается пунктиром).
_Советы:_
Используйте ленивую конъюнкцию.
Если в операторе if стоит return, то else уже не нужен.
```suggestion
        {
            if (y >= 0.5 && y <= 1.5 && x <= 2)
                return true;
                
            if (x > 2)
                return true;

            return false;
        }
```
Логическое выражение и, соответственно, оператор условия можно было упростить:
```suggestion
        {
            if (x >= 2 || y >= 0.5 && y <= 1.5 )
                return true;

            return false;
        }
```